cfgmgr/sflowmgr.cpp

```diff
@@ -106,13 +110,19 @@ void SflowMgr::sflowUpdatePortInfo(Consumer &consumer)
                 speed_change = true;
             }
 
+            string def_dir = "rx";
+            if (m_sflowPortConfMap[key].dir != def_dir)
+            {
+                m_sflowPortConfMap[key].dir = def_dir;
```

Comment:
Which means user has configured direction, please updated port_info.local_dir_cfg as true.

Reply:
> Which means user has configured direction, please updated port_info.local_dir_cfg as true.

I will change the check as below. Intention is to update default value "rx" when port_info is initialized for first time as it was updated with empty string at line 94.
```
  if (m_sflowPortConfMap[key].dir != def_dir && !port_info.local_dir_cfg )
 {
     m_sflowPortConfMap[key].dir = def_dir;
 }
```
If user has configured the direction and if port speed is changed, in this flow the existing user set direction must be used.